frontend/test/metabase/scenarios/collections/permissions.cy.spec.js

```diff
@@ -366,9 +366,12 @@ describe("collection permissions", () => {
               cy.signIn(user);
             });
 
-            it("should not be offered to duplicate dashboard in collections they have `read` access to", () => {
+            it("should be offered to duplicate dashboard in collections they have `read` access to", () => {
               cy.visit("/collection/root");
-              findEllipsisMenuFor("Orders in a dashboard").should("not.exist");
+              openEllipsisMenuFor("Orders in a dashboard");
+              popover()
+                .findByText("Duplicate this item")
+                .should("exist");
```

Comment:
```suggestion
              cy.findByText("Duplicate this item");
```

Since there is no other string like this on the page, we don't have to explicitly check that it's in popover. You don't have to fix it now, since it seems we'll have to re-group and tidy up all tests in this file. Just a not for the future.

Reply:
Thanks, I'll follow this approach from now 👌Merging the PR for now